<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tes4</class>
 <widget class="QWidget" name="tes4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tes4</string>
  </property>
  <widget class="QGroupBox" name="boxPrimary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Primary</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>70</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbColumn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="currentText">
     <string>None</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbOrder">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>32</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>61</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdv">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Advance</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="boxSecondary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Secondary</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>70</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbColumn_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="currentText">
     <string>None</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbOrder_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="boxTertiary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Tertiary</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>70</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbColumn_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="currentText">
     <string>None</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbOrder_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ascending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Descending</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdv</sender>
   <signal>toggled(bool)</signal>
   <receiver>boxSecondary</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdv</sender>
   <signal>toggled(bool)</signal>
   <receiver>boxTertiary</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
